### PR TITLE
Add site-specific docs

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

---

**Related issue :**

### CODE REVIEW

**Comment by enolfc** [comment]
@thebe14 how this fits into the review of the documentation you were planning?

**Comment by gwarf** [comment]
Even if some changes are planned/discussed I think it would be good to already include this content, it can be moved/consolidated later, when new things will be pushed.

**Comment by sebastian-luna-valero** [comment]
Thanks.

Would it also be worth circulating the PR in `fedcloud-tf` emailing list to ask other cloud providers to submit their url for site specific docs?

**Comment by gwarf** [comment]
I'm not sure about the location of this page, currently it's under Compute, but I feel it may be easier to find it at the same level as/near the community-specific documentation. Also it could include info about other non compute services running at a given site.

**Comment by gwarf** [comment]
While at it, I guess it may also be needed to use sudo to edit docker's config file.

**Comment by gwarf** [comment]
Another point I didn't caught earlier: the command lines in the shell excerpts should start with a prompt, like `$ sudo vi /etc/docker/daemon.json`

File: content/en/users/service-providers/_index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Site-specific documentation for Service Providers
```
Comment (gwarf) [inline content/en/users/service-providers/_index.md:2]:
```suggestion
title: Service provider specific documentation for Users
```

File: content/en/users/service-providers/_index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+linkTitle: Service provider specific documentation
+type: docs
+weight: 100
+description: >
+  Documentation provided by each cloud provider in the EGI Federation.
```
Comment (gwarf) [inline content/en/users/service-providers/_index.md:7]:
```suggestion
  Documentation specific to the individual services providers.
```
Or something around this idea but better phrased.

File: content/en/users/service-providers/_index.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+Some cloud providers in the EGI Federation publish documentation about how
```
Comment (gwarf) [inline content/en/users/service-providers/_index.md:10]:
```suggestion
Some service providers in the EGI Federation publish documentation about how
```

File: content/en/users/service-providers/_index.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+to work with their site. Below is a list:
+
+- [CESNET-MCC](https://docs.cloud.muni.cz/)
+- [IFCA-LCG2](https://confluence.ifca.es/display/IC/Computing+Resources)
+- [IN2P3-IRES](https://scigne.fr/en/user-support/online-documentation/)
+
+This information is handy to deal with the particularities of each OpenStack
+deployment.
```
Comment (gwarf) [inline content/en/users/service-providers/_index.md:18]:
```suggestion
This information is handy to deal with the particularities of each services deployed by a given provider. It usually covers OpenStack deployments, but may also document other services.
```

**Review by gwarf — APPROVED** [review]
LGTM as an initial way to share information about this, thanks!

**Comment by gwarf** [comment]
> Would it also be worth circulating the PR in `fedcloud-tf` emailing list to ask other cloud providers to submit their url for site specific docs?

Not sure if we should wait before merging it, but I think it would be good to at least report it during a fedcloud meeting, so that providers can send link they may have, it can easily be added after this one is merged.

**Review by enolfc — APPROVED** [review]
Go for it

**Pull request merged**